src/commands/hello/org.ts

```diff
@@ -53,10 +53,10 @@ export default class Org extends SfdxCommand {
   protected static requiresProject = false;
 
   public async run(): Promise<AnyJson> {
-    const name = this.flags.name || 'world';
+    const name: string = this.flags.name || 'world';
 
     // this.org is guaranteed because requiresUsername=true, as opposed to supportsUsername
-    const conn = this.org!.getConnection();
```

Comment:
This do something like [this](https://github.com/forcedotcom/salesforcedx-apex/pull/64/files#diff-090d9fc625ac790f63e2a2b9a0ad93fdR33) to fix it.